operator/gitops/argocd/pipeline-service/pipelines-as-code/kustomization.yaml

```diff
@@ -28,7 +28,7 @@ patchesJson6902:
       - op: replace
         path: "/spec/template/spec/containers/0/image"
         value:
-          registry.redhat.io/openshift-pipelines/pipelines-pipelines-as-code-rhel8:v1.10.0-6
+          registry.redhat.io/openshift-pipelines/pipelines-pipelines-as-code-rhel8:v1.9.2-1
```

Comment:
using tags here is nice, isn't it @Roming22 ;-)

Reply:
:rofl: